sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosBulkExecutionOptions.java

```diff
@@ -257,6 +267,17 @@ public int getMaxMicroBatchConcurrency(CosmosBulkExecutionOptions options) {
                     return options.getMaxMicroBatchConcurrency();
                 }
 
+                @Override
+                public Integer getMaxConcurrentCosmosPartitions(CosmosBulkExecutionOptions options) {
+                    return options.getMaxConcurrentCosmosPartitions();
+                }
+
+                @Override
+                public CosmosBulkExecutionOptions setMaxConcurrentCosmosPartitions(
```

Comment:
Is this public? Should it be? For .NET we didn't want to leak any notion of physical partitions

Reply:
This isn't public - it is the cross package helper (which is animplementation package API)

Reply:
Makes sense for internal consumption (from Spark) but not sure about totally public. @kirankumarkolli and @kushagraThapar might want to weigh in?

Reply:
Again - it isn't public. These methods implement an interface form the implementation package - and the implementing class is internal API.